QgisModelBaker/gui/panel/pg_config_panel.py

```diff
@@ -440,3 +447,45 @@ def _keep_custom_settings(self):
             self.pg_ssl_mode_combo_box.currentData(),
             PgConfigPanel._SERVICE_COMBOBOX_ROLE.SSLMODE,
         )
+
+    def _fill_schema_combo_box(self):
```

Comment:
When is this called? Is this called when I change the e.g. db name?

Reply:
Yes, this is the idea (but I have completely forgotten it in the end 🙈)
I will add a timer so that it isn't triggered continuosly during editation. Like it is done here https://github.com/opengisch/QgisModelBaker/blob/203428af241f25b956fe5418e1c95269d832087a/QgisModelBaker/gui/dataset_manager.py#L78